src/transformers/__init__.py

```diff
@@ -609,6 +609,16 @@
         "SpecialTokensMixin",
         "TokenSpan",
     ],
+    "tools": [
+        "Agent",
```

Comment:
```suggestion
```

Reply:
To limit breaking / design changes maybe it might be good to not put too much in the public API?

Reply:
I'd keep it so users can define their custom Agent.